src/components/shop-common-style.js

```diff
@@ -0,0 +1,37 @@
+import { html } from '../../node_modules/@polymer/lit-element/lit-element.js';
+
```

Comment:
What do you think about calling this file and exporting `ShopSharedStyles` (also with an uppercase cause it's like...A Whole Thing. I don't know.

Reply:
I'm open to changing the name, but not going to do anything until we figure out the blessed way to efficiently share CSS between elements.